include/deal.II/simplex/grid_generator.h

```diff
@@ -27,221 +27,225 @@
 
 DEAL_II_NAMESPACE_OPEN
 
-namespace Simplex
+namespace GridGenerator
 {
   /**
-   * This namespace provides a collection of functions to generate simplex
-   * triangulations for some basic geometries.
+   * Create a coordinate-parallel brick from the two diagonally opposite
+   * corner points @p p1 and @p p2. The number of vertices in coordinate
+   * direction @p i is given by <tt>repetitions[i]+1</tt>.
    *
-   *  @ingroup simplex
+   * @note This function connects internally 4/8 vertices to quadrilateral/
+   *   hexahedral cells and subdivides these into 2/5 triangular/
+   *   tetrahedral cells.
```

Comment:
This is going to create no space before but one space after the `/` when rendered. Just put the first word of lines 38 and 39 at the end of the previous line and then let indentation break where necessary.